skimage/measure/tests/test_regionprops.py

```diff
@@ -184,12 +185,33 @@ def test_equiv_diameter():
 
 def test_euler_number():
     en = regionprops(SAMPLE)[0].euler_number
-    assert en == 1
+    assert en == 0
```

Comment:
why did you have to change the value here and below?

Reply:
Sorry about that, I made a mistake in writing the test.

Reply:
Sorry about that, I made a mistake in writing the test.